packages/components/src/base-control/style.scss

```diff
@@ -18,6 +18,6 @@
 	.components-base-control__help {
 		margin-top: -$grid-size;
 		font-style: italic;
-		margin-bottom: 0;
+		margin-bottom: 20px;
```

Comment:
Thanks for the PR @geekpulp 

I'm wondering if the best way to fix is to add a margin bottom to the help component (which may or may not be present depending on what component is used). It feels like the root issue is that there's no space between controls. So maybe the fix is more to target something like `.components-base-control + .components-base-control`

What do you think?

Reply:
Another thing is I think we're trying to make sure we use the $grid-size as a basis for all these spaces in the UI. So maybe `$grid-size*2` or something? 

cc @jasmussen

Reply:
Thanks for the feedback. You’re right. I’ll change my approach and update.

Reply:
@youknowriad Ok, I've had another crack at it. Seems to have done the trick. Also updated a related stylesheet to use $grid-size.

It's quite a learning curve but this is strangely fun.

Reply:
Thank you for using $grid-size! If you need to, there's also $grid-size-large, which is the same as $grid-size * 2.

Reply:
@jasmussen Cool, I'll change that then. Is there somewhere I can go to see standards like grid-size so I can proactively make changes like that? Is it something documented somewhere?

Reply:
Hmm showing my newbie skills here with Git. Sorting it.